src/pages/AdminCircleApiPage/ApiKeyForm.tsx

```diff
@@ -0,0 +1,189 @@
+      ...data,
+      circle_id: circleId,
+    });
+
+  const { field: name } = useController({
+    name: 'name',
+    control,
+    defaultValue: '',
+  });
+
+  const { field: readCircle } = useController({
+    name: 'read_circle',
+    control,
+    defaultValue: false,
+  });
+
+  const { field: readEpochs } = useController({
+    name: 'read_epochs',
+    control,
+    defaultValue: false,
+  });
+
+  const { field: readMemberProfiles } = useController({
+    name: 'read_member_profiles',
+    control,
+    defaultValue: false,
+  });
+  const { field: readPendingTokenGifts } = useController({
+    name: 'read_pending_token_gifts',
+    control,
+    defaultValue: false,
+  });
+  const { field: readNominees } = useController({
+    name: 'read_nominees',
+    control,
+    defaultValue: false,
+  });
+
+  return (
+    <Form
+      onSubmit={handleSubmit(onSubmit)}
+      css={{
+        width: '100%',
+        alignItems: 'center',
+        display: 'flex',
+        flexDirection: 'column',
+      }}
+    >
+      <Text font="source" size="medium">
+        Circle API keys allow for third party apps to read data from and
+        interact with your circle. You can configure specific permissions for
+        each key.
+      </Text>
+      <FormLabel htmlFor="name" type={'textField'} css={{ mt: '$lg' }}>
+        Label
+      </FormLabel>
+      <TextField
+        css={{ width: '100%' }}
+        id="name"
+        placeholder={'What is this API key for?'}
+        {...name}
+      />
+      <Box
+        css={{
+          display: 'flex',
+          gap: '$sm',
+          my: '$lg',
+          flexDirection: 'column',
+          width: '100%',
+        }}
+      >
+        <CheckBox
+          {...readCircle}
+          label={API_PERMISSION_LABELS['read_circle']}
+          infoTooltip={
+            <>Allows reading your circle name, logo, and configuration.</>
+          }
+        />
+        <CheckBox
+          {...readEpochs}
+          label={API_PERMISSION_LABELS['read_epochs']}
+          infoTooltip={<>Allows reading information about historical epochs.</>}
+        />
+        <CheckBox
+          {...readMemberProfiles}
+          label={API_PERMISSION_LABELS['read_member_profiles']}
+          infoTooltip={
+            <>
+              Allows reading member profiles (name, socials, avatars), excluding
+              wallet addresses.
+            </>
+          }
+        />
+        <CheckBox
+          {...readPendingTokenGifts}
+          label={API_PERMISSION_LABELS['read_pending_token_gifts']}
+          infoTooltip={
+            <>
+              Allows reading pending allocations in the current epoch (excluding
+              notes).
+            </>
+          }
+        />
+        <CheckBox
+          {...readNominees}
+          label={API_PERMISSION_LABELS['read_nominees']}
+          infoTooltip={<>Allows reading information about circle nominees.</>}
+        />
+      </Box>
+      <Button
```

Comment:
I think this button would look better taller, like on the Allocation flow , which is ```<UIButton
            size="large"
```